crates/ra_proc_macro/src/rpc.rs

```diff
@@ -34,15 +34,15 @@ pub struct ListMacrosResult {
 pub struct ExpansionTask {
     /// Argument of macro call.
     ///
-    /// In custom derive that would be a struct or enum; in attribute-like macro - underlying
+    /// In custom derive this will be a struct or enum; in attribute-like macro - underlying
     /// item; in function-like macro - the macro body.
     #[serde(with = "SubtreeDef")]
     pub macro_body: Subtree,
 
-    /// Names of macros to expand.
+    /// Names of macros to expand. // TODO: are they comma-separated?
```

Comment:
Small question here @edwin0cheng

Reply:
Ouch, should be "Name of macro to expand"

Reply:
I have doubts because the next line says:
> In custom derive those are names of derived traits (`Serialize`, `Getters`, etc.).

So do we pass create a separate `ExpansionTask` for each derived trait in fact?

Reply:
Yes, we separate ExpansionTask for each derived. I forget to change the comment when I copy the code from.